lightningd/options.c

```diff
@@ -477,23 +477,23 @@ static const struct config testnet_config = {
 	/* Testnet sucks */
 	.ignore_fee_limits = true,
 
-	/* Rescan 5 hours of blocks on testnet, it's reorg happy */
-	.rescan = 30,
+	/* Rescan 1 hours of blocks on testnet, it's reorg happy */
+	.rescan = 60,
 
 	/* Fees may be in the range our_fee - 10*our_fee */
 	.max_fee_multiplier = 10,
 
-	.use_dns = true,
+	.use_dns = false,
 };
 
 /* aka. "Dude, where's my coins?" */
-static const struct config mainnet_config = {
+static const struct config groestlcoin_mainnet_config = {
```

Comment:
Since we are plain c-grs-lightning, we should move back using to mainnet_config to make it easier to merge with upstream